### PR TITLE
Combined AcceptApplication, RejectApplication, and GetApplication

### LINKED ISSUE #112: Combine usecases
Please combine closely related usecases into one. For example, create, reject, and accept applications could be combined into one usecase: manageapplications. All files can remain the same, but their controllers should be combined into one, called ManageApplicationsController.

### PR DESCRIPTION
**Pull request by tArthur1**
Combined AcceptApplication, RejectApplication, and GetApplication into ManageApplications use_case

### CODE REVIEW

**Review by sherwinokhowat — CHANGES_REQUESTED** [review]
Please do it for projects and users as well. That is, combine them into manageprojects and manageusers usecases.

**Comment by Raqkkkkk** [comment]
Also we might need to update tests after this.

**Comment by tArthur1** [comment]
Please review my pull request, I've combined use cases for project and users into ManageProjects and ManageUsers respectively.

File: src/main/java/Main.java [hunk 1]
```diff
@@ -53,10 +57,14 @@ public static void main(String[] args) {
         ViewManagerModel viewManagerModel = new ViewManagerModel();
         ViewManager viewManager = new ViewManager(views, cardLayout, viewManagerModel);
 
-        // Create User Panel
+        // Manage Users
         CreateUserPanelViewModel createUserPanelViewModel = new CreateUserPanelViewModel();
+        EditProfileViewModel editProfileViewModel = new EditProfileViewModel();
+        ManageUsersController manageUsersUseCaseFactory = ManageUsersUseCaseFactory.create(createUserPanelViewModel, editProfileViewModel);
```
Comment (Raqkkkkk) [inline src/main/java/Main.java]: I think this variable could be changed to another name to make it more clear what it is (like manageUsersController).

File: src/main/java/usecase/manageusers/ManageUsersController.java
Comment (Raqkkkkk) [inline src/main/java/usecase/manageusers/ManageUsersController.java:1]: remember to delete commented out code if it is no longer needed

**Review by Raqkkkkk — CHANGES_REQUESTED** [review]
overall seems to only have some minor formatting and style issues
functionality should be fine if the tests still pass

**Review by sherwinokhowat — CHANGES_REQUESTED** [review]
Please move the associated packages to the combined use case package. For example, the create application package should be nested inside of the manageapplications package. This applies to the rest of the combined usecases as well.

**Comment by tArthur1** [comment]
I moved each use case into their respective manage use case, please review my request.

**Comment by Raqkkkkk** [comment]
Dealt with requested changes during merge

**Pull request merged**